### PR TITLE
MULE-9925 - Remove default app from distribution

### PR DESCRIPTION
**Pull request by pablolagreca**

### CODE REVIEW

File: distributions/standalone/assembly-whitelist.txt [hunk 1]
```diff
@@ -14,20 +14,6 @@
 /mule-standalone-${productVersion}/MIGRATION.txt
 
 #
-# /apps
-#
-/mule-standalone-${productVersion}/apps/default
```
Comment (pablokraan) [inline distributions/standalone/assembly-whitelist.txt]: Should not include /mule-enterprise-standalone-${productVersion}/apps in the list?

File: distributions/standalone/src/main/resources/apps/IGNORE [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Git does not allow versioning of an empty folder so we add this file here to create the folder in git.
```
Comment (pablokraan) [inline distributions/standalone/src/main/resources/apps/IGNORE]: This file should not go into the distribution. If there is not way to remove the file and keep the folder in the distro, the rename the file to readme.txt of something like that

Comment (pablolagreca) [inline distributions/standalone/src/main/resources/apps/IGNORE]: This file doesn't go into the distro. Is the same as we do in domains. We put it here so git can track the folder but then we ignore it in assembly.xml

File: distributions/standalone/assembly-whitelist.txt (hunk 1 above)
Comment (pablolagreca) [inline distributions/standalone/assembly-whitelist.txt]: Is in the list above

**Pull request merged**